src/components/TileGrid/Tile/RectTile.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import styled from "styled-components";
+import { ReactComponent as EduLogo } from "../../../assets/svgs/edu.svg";
+import one from "../../../assets/images/flyer.png";
+import two from "../../../assets/images/arduino.png";
+import three from "../../../assets/images/cscareers.png";
+import four from "../../../assets/images/donuts.png";
+import five from "../../../assets/images/interview.png";
+import six from "../../../assets/images/sweacmfinal.png";
+
+interface propTypes {
+  title: string;
+  date: string;
+  time: string;
+  num: number;
+}
+
+const RectTile = (props: propTypes) => {
+  return (
+    <TileComponent tabIndex={props.num}>
+      <div className="tile-card">
+        <div className="tile-content">
+          <div className="tile-top">
+            <div className="tile-title">{props.title}</div>
+          </div>
+          <div className="tile-end">
+            <div className="tile-end-left">
+              <div className="tile-date">{props.date}</div>
+              <div className="tile-time">{props.time}</div>
+            </div>
+            <div className="tile-end-right">
+              <EduLogo />
+            </div>
+          </div>
+        </div>
+      </div>
+    </TileComponent>
+  );
+};
+
+//the next few lines are temporary, just to get a cool effect even though its hardcoded
+//absolutely will get refactored
+let arrayOfImages = [one, two, three, four, five, six];
+
+const TileComponent = styled.div`
+  .tile-card {
+    width: 230px;
+    height: 400px;
+    border-radius: 30px;
+    background:url(${props => arrayOfImages[props.tabIndex ? props.tabIndex : 1]}); 
```

Comment:
We should probably make the default image be the respective division logo or just a generic acm logo in case someone doesn't provide one when creating an event. We should also find a better way to pass the prop to the styled div since it exists outside the component.